pkg/forwarder/interceptor/interceptor.go

```diff
@@ -103,12 +104,14 @@ func (i *interceptor) CreateContainer(ctx context.Context, req *pb.CreateContain
 					logger.Printf("Failed to create dir: %v", err)
 				}
 			}
-
-			if isTargetPath(m.Source, volumeTargetPath) {
-				logger.Printf("Waiting for device mounted to: %s", m.Source)
-				err := waitForDeviceMounted(ctx, m.Source)
-				if err != nil {
-					return nil, err
+			volumeTargetPathSlice := strings.Split(volumeTargetPath, ",")
```

Comment:
It's better to split the slice outside the for, and calling strings.Trim() to clean the spaces before calling isTargetPath